pkg/token/token_test.go

```diff
@@ -347,11 +346,6 @@ func TestFormatJson(t *testing.T) {
 			EnvKey:           "KUBERNETES_EXEC_INFO",
 			ExpectApiVersion: clientauthv1beta1.SchemeGroupVersion.String(),
 		},
-		{
-			Name:             "KUBERNETES_EXEC_INFO with v1alpha1",
-			EnvKey:           "KUBERNETES_EXEC_INFO",
-			ExpectApiVersion: clientauthv1alpha1.SchemeGroupVersion.String(),
```

Comment:
keep the test but replace this with client.authentication.k8s.io/v1alpha1.

If there is further doubt about whether simply updating client-go could affect authenticator's compatibility to client.authentication.k8s.io/v1alpha1 i suggest someone implement in a follow-up PR an e2e test to validate kubeconfigs with all versions, v1alpha1 v1beta1, v1. (Could be me, IDK, I'm not volunteering at this time just suggesting)